From c5ca0504d7e53830db64ae03b3b20ef682da304c Mon Sep 17 00:00:00 2001
From: Kumar Appaiah <a.kumar@alumni.iitm.ac.in>
Date: Wed, 8 Aug 2012 15:33:41 -0500
Subject: [PATCH 2/2] Remove unneeded "id" field for HTML tag when reporting a
 problem

The <pre> tag used to report problems reused the "problem-%d" id, thus
causing the generated HTML to be invalid. This commit fixes that
error.
---
 library.php |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/library.php b/library.php
index 25c2ef5..3a0e238 100644
--- a/library.php
+++ b/library.php
@@ -954,7 +954,7 @@ function buildd_failures($problems, $pas, $suite) {
           $archs = make_list($archs_data);
         }
 	$message = detect_links(htmlentities($message));
-	printf("<p><b>%s for <a href=\"package.php?p=%s&amp;suite=%s\">%s</a> on %s:</b></p>\n<pre id=\"problem-%d\" class=\"failure\">%s%s</pre>\n",
+	printf("<p><b>%s for <a href=\"package.php?p=%s&amp;suite=%s\">%s</a> on %s:</b></p>\n<pre class=\"failure\">%s%s</pre>\n",
                ucfirst($reason),
                urlencode($package),
                $suite,
-- 
1.7.10.4

